MID Server/MID Server Script Includes/IllumioAuthenticateToPCE.js

```
@@ -28,10 +28,17 @@ IllumioAuthenticateToPCE.prototype = {
         this.utils = new IllumioPCEUtils(this.timeZone);
         this.protocol = this.utils.getPortFromUrl(this.pceUrl);
 
+        this.retryParams = null;
```

This null default doesn't really make sense - if the parsing fails and we pass null to the `IllumioHTTPClient` init then it will fall over on trying to access the object members anyway. We should either bubble the exception here or better yet pass default values for the retry params.

---

Will resolve this